main.js

```diff
@@ -160,4 +160,125 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//////////////////////////// Opdracht èèn /////////////////////////////////////
+
+const televisionNames = inventory.map((television) => {
+  return television.name;
+});
+
+console.log(televisionNames);
+
+//
+
+const televisionsInStock = inventory.filter((stock) => {
+  return stock.originalStock === stock.sold;
+});
+
+console.log(televisionsInStock);
+
+//
+
+const withAmbilight = inventory.filter((ambilight) => {
+  return ambilight.options.ambiLight;
+});
+
+console.log(withAmbilight);
+
+//
+
+const pricingOrder = inventory.sort((a, b) => {
+  if (a.price >= b.price) {
+    return 1;
+  }
+    else if (a.price <= b.price) {
+      return -1
+    }
+      else return 0
```

Comment:
Dit kun je nog wat versimpelen
```suggestion
  return a.price - b.price;
```